### PR TITLE
Add Place v3

### PR DESCRIPTION
**Pull request by VincentRbbmnd**
New version (3) of Place to support the realization of TrailerVote/trailervote-api#484

### CODE REVIEW

**Comment by VincentRbbmnd** [comment]
Blocked until TrailerVote/trailervote-fixtures#12 is merged and used in this repo

**Comment by thexa4** [comment]
Merged the blocking request.

File: ruby/lib/trailer_vote/media_types/place.rb
Comment (thexa4) [inline ruby/lib/trailer_vote/media_types/place.rb]: Children link cannot be nil right? There should always be a resource that shows all the children, it might be empty but it cannot not exist.

**Comment by karensg** [comment]
Shall I add ts definition to here as well?

File: ruby/test/trailer_vote/media_types/place_test.rb
Comment (thexa4) [inline ruby/test/trailer_vote/media_types/place_test.rb]: Shouldn't it be v3_json?

File: ruby/test/trailer_vote/media_types/place_test.rb
Comment (thexa4) [inline ruby/test/trailer_vote/media_types/place_test.rb:24]: Shouldn't there also be a create one?

**Review by thexa4 — APPROVED** [review]

**Pull request merged**
